django_airavata/apps/workspace/static/django_airavata_workspace/js/entry-user-storage.js

```diff
@@ -6,7 +6,7 @@ import VueRouter from "vue-router";
 
 const routes = [
   {
-    path: "*",
+    path: "/*",
```

Comment:
Can you explain why this change was made?

Reply:
This can be removed